src/modules/displayComments.js

```diff
@@ -0,0 +1,12 @@
+import commentsCounter from './commentsCounter.js';
+import fetchComment from './fetchcomments.js';
+
+const displayComments = async (id) => {
+  const comments = await fetchComment(id);
+  console.log(comments);
```

Comment:
- [ ] as for the [ JS BEST PRACTICES](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) , please do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website. kindly remove it 🙏